src/prymitive.ts

```diff
@@ -0,0 +1,120 @@
+class School {
+  directions:Direction[] = [];
+
+  addDirection(direction:Direction) {
+    this.directions.push(direction);
+  }
+}
+
+class Direction {
+  private _name: string;
+  levels:Level[]  = [];
+
+  constructor( name:string ) {
+    this._name = name;
+  }
+  get name():string {
+    return this._name;
+  }
+
+  addLevel(level:Level) {

```

Comment:
Please, add explicitly type annotation to the value that returns `addLevel `function.